gcp-vertex-ai-mojo-scorer/src/main/java/ai/h2o/mojos/deploy/gcp/vertex/ai/controller/ModelsApiController.java

```diff
@@ -56,7 +56,9 @@ public ResponseEntity<ScoreResponse> getScore(
       ScoreRequest request = getRestScoreRequest(gcpRequest);
       ScoreResponse response = getGcpScoreResponse(scorer.score(request));
       // return ResponseEntity.ok(scorer.score(request));
-      return ResponseEntity.ok(response);
+      return ResponseEntity.ok().contentLength(
+              new ObjectMapper().writeValueAsString(response).length()
+      ).body(response);
```

Comment:
Avoid serializing twice.
```suggestion
      final String responsePayload = new ObjectMapper().writeValueAsString(response);
      return ResponseEntity.ok()
              .contentLength(responsePayload.length())
              .body(responsePayload);
```

Reply:
can't be a string, need to be an ScoreResponse